primitives/tf_encrypted/primitives/sodium/cc/ops.cc

```diff
@@ -16,7 +16,7 @@ REGISTER_OP("SodiumEasyBoxGenNonce")
     .SetIsStateful();
 
 REGISTER_OP("SodiumEasyBoxSealDetached")
-    .Input("plaintext: uint8")
+    .Input("plaintext: float32")
```

Comment:
check out https://github.com/tf-encrypted/tf-big/blob/master/tf_big/cc/kernels/big_kernels.cc#L31-L47 and https://github.com/tf-encrypted/tf-big/blob/master/tf_big/cc/kernels/big_kernels.cc#L303-L304 for how to support `uint8` and `float32` at the same time